From a55cc467046c64188830f230af5304e7e2212ee0 Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Thu, 6 Jun 2013 15:11:48 +0200
Subject: [RHEL6 LIBREPORT PATCH 95/95] fixed relro flags rhbz#812283

Signed-off-by: Jiri Moskovcak <jmoskovc@redhat.com>
Signed-off-by: Jakub Filak <jfilak@redhat.com>

Conflicts:
	src/client-python/Makefile.am
	src/report-python/Makefile.am
---
 src/client-python/Makefile.am | 6 +++---
 src/report-python/Makefile.am | 6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/client-python/Makefile.am b/src/client-python/Makefile.am
index 4048d56..4da3aa6 100644
--- a/src/client-python/Makefile.am
+++ b/src/client-python/Makefile.am
@@ -19,11 +19,11 @@ _reportclient_la_CPPFLAGS = \
     $(GLIB_CFLAGS) \
     $(PYTHON_CFLAGS) \
     -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror \
-    -fPIE -Wl,-z,relro,now
+    -fPIE
 _reportclient_la_LDFLAGS = \
     -module \
     -avoid-version \
-    -export-symbols-regex init_reportclient
+    -export-symbols-regex init_reportclient \
+    -Wl,-z,relro -Wl,-z,now
 _reportclient_la_LIBADD = \
     ../lib/libreport.la
diff --git a/src/report-python/Makefile.am b/src/report-python/Makefile.am
index 93ee625..e961eba 100644
--- a/src/report-python/Makefile.am
+++ b/src/report-python/Makefile.am
@@ -23,12 +23,12 @@ _pyreport_la_CPPFLAGS = \
     $(GLIB_CFLAGS) \
     $(PYTHON_CFLAGS) \
     -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror \
-    -fPIE -Wl,-z,relro,now
+    -fPIE
 _pyreport_la_LDFLAGS = \
     -module \
     -avoid-version \
-    -export-symbols-regex init_pyreport
+    -export-symbols-regex init_pyreport \
+    -Wl,-z,relro -Wl,-z,now
 _pyreport_la_LIBADD = \
     ../lib/libreport.la
 
-- 
1.8.2.1

